topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.md

```diff
@@ -0,0 +1,910 @@
+> >   - Some bioinformaticians would recommend filtering each sample individually, but this is difficult in larger scale and in this case (you're welcome to give it a go! You'd have to filter separately and then concatenate), it won't make a notable difference in the final interpretation.
+> >
+> {: .solution}
+>
+> What threshold should you set for `log1p_total_counts`?
+> 1. Which plot(s) addresses this?
+> 2. What number would you pick?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. As before, any plot with `log1p_n_total_counts` will do! Again, we'll use a scatterplot here, but you can use a violin plot if you wish!
+> > ![Scatter-countsxmito](../../images/wab-scatter-countsxmito.png "Scatterplot - mito x UMIs (Raw)")
+> >
+> > 2. We can see that we will need to set a higher threshold (which makes sense, as you'd expect more UMI's per cell rather than unique genes!). Again, perhaps being a bit aggressive in our threshold, we might choose `6.3`, for instance (which amounts to around 500 counts/cell).
+> >   - In an ideal world, you'll see a clear population of real cells separated from a clear population of debris. Many samples, like this one, are under-sequenced, and such separation would likely be seen after deeper sequencing!
+> >
+> {: .solution}
+>
+> What threshold should you set for `pct_counts_mito`?
+> 1. Which plot(s) addresses this?
+> 2. What number would you pick?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Any plot with `pct_counts_mito` would do here, however the scatterplots are likely the easiest to interpret. We'll use the same as last time.
+> > ![Scatter-countsxmito](../../images/wab-scatter-countsxmito.png "Scatterplot - mito x UMIs (Raw)")
+> >
+> > 2. We can see a clear trend wherein cells that have around 5% mito counts or higher also have far fewer total counts. These cells are low quality, will muddy our data, and are likely stressed or ruptured prior to encapsulation in a droplet. While 5% is quite a common cut-off, this is quite messy data, so just for kicks we'll go more aggressive with a `4.5%`.
+> >    - In general, you must adapt all cut-offs to your data - metabolically active cells might have higher mitochondrial RNA in general, and you don't want to lose a cell population because of a cut-off.
+> >
+> {: .solution}
+{: .question}
+
+## Apply the Thresholds
+
+It's now time to apply these thresholds to our data! First, a reminder of how many cells and genes are in your object: `25281 cells` and `35734 genes`. Let's see how that changes each time!
+
+> ### {% icon details %} Working in a group? Decision-time!
+> If you are working in a group, you can now divvy up a decision here with one *control* and the rest varied numbers so that you can compare results throughout the tutorials.
```

Comment:
"divvy up" maybe simplify phrases like this, as I'm not sure how well they translate